.pylintrc

```diff
@@ -1,2 +1,3 @@
 [MESSAGES CONTROL]
 good-names=x,y,xs,ys,n,i,j
+disable=eval-used,too-many-return-statements,too-many-locals,protected-access
```

Comment:
:cowboy_hat_face:

Reply:
The only way to remove these is to rewrite DeepDiff from scratch ;)